adapters/flipp/flipp.go

```diff
@@ -32,8 +37,17 @@ type adapter struct {
 	endpoint string
 }
 
+type UUIDGenerator struct {
+	Generate func() (uuid.UUID, error)
+}
+
+func Generate() (uuid.UUID, error) {
+	return uuid.NewV4()
+}
+
```

Comment:
could make use of uuid helper

https://github.com/prebid/prebid-server/blob/24abd9c08981a49e9621ec34c05289e187f19a81/util/uuidutil/uuidutil.go#L7-L19

Reply:
@onkarvhanumante thank you, using uuid helper now